rd-cpp/export/rd-cpp.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>JetBrains.RdFramework.Cpp</id>
+        <authors>JetBrains</authors>
+        <version>0.0.0.0</version>
+        <repository type="git" url="https://github.com/JetBrains/rd" />
+        <description>JetBrains Rd Framework CPP implementation</description>
+        <requireLicenseAcceptance>true</requireLicenseAcceptance>
```

Comment:
While this shouldn't matter much, it's better to set this to `false`. `requireLicenseAcceptance=true` is usually reserved for cases when some unusual EULA is used. In our case, it's the standard Apache license.

But again, this doesn't matter much, because our main use cases don't include installing these packages via the standard NuGet client.